src/LanguageSearch.php

```diff
@@ -75,6 +79,25 @@ private function getPostLang( $post_args, $post_id ) {
 		return $lang;
 	}
 
+	private function getPostLangAsTranslated( $post_args, $post_id ) {
+		$active_languages = array_keys( $this->sitepress->get_active_languages() );
```

Comment:
@decodekult You might want to use `wpml_element_type` instead of prepending 'post_':
https://wpml.org/wpml-hook/wpml_element_type/

Reply:
Done, thanks!